packages/webcomponents/src/components/business-forms/payment-provisioning/payment-provisioning.scss

```diff
@@ -56,6 +56,13 @@
   }
 }
 
+input[type='text'][disabled] {
```

Comment:
I believe this should be handled by bootstrap - do we know why it's not?

Reply:
I'm not personally sure why this rule was written. It used to exist in the scss file for `form-control-text`, which I deleted the file and moved this rule to the top level scss file, `payment-provisioning.scss`

Here is the disabled text input with the bootstrap default styles (removing this rule declaration for `input`):

![Screenshot 2024-06-27 at 2 42 41 PM](https://github.com/justifi-tech/web-component-library/assets/87510749/008838c5-886d-42a0-9aa0-f76647602079)

And, here is the disabled text input with the styles that someone wrote for this input, and the way it has looked for quite a while. 

![Screenshot 2024-06-27 at 2 44 08 PM](https://github.com/justifi-tech/web-component-library/assets/87510749/ada148ff-d473-4477-906f-926e154e7506)


I guess I'm fine either way, what do you think?

Reply:
I may have done that myself, but we should use the bootstrap default and remove instances of rolling our own disabled styles